test/org/sagebionetworks/bridge/services/AuthenticationServiceImplTest.java

```diff
@@ -50,6 +51,8 @@
 public class AuthenticationServiceImplTest {
```

Comment:
Tests for checking consent statuses in the user session in signIn and verifyEmail?